app/uk/gov/hmrc/plasticpackagingtax/registration/views/liability/tax_start_date_page.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import java.time.LocalDate
+@import uk.gov.hmrc.plasticpackagingtax.registration.controllers.liability.{routes => pptRoutes}
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.registration.Registration
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.request.JourneyRequest
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.models.{BackButton, Title}
+
+@this(
+ formHelper:        FormWithCSRF,
+ govukLayout:       main_template,
+ pageHeading:       pageHeading,
+ paragraphBody:     paragraphBody,
+ continueButton:    continue,
+ sectionHeader:     sectionHeader
+)
+@(startDate: LocalDate, hasExceededThresholdWeight: Boolean, backLink: Call)(implicit request: JourneyRequest[_], messages: Messages)
+
+@govukLayout(
+    title = Title("liability.taxStartDate.title"),
+    backButton = Some(BackButton(messages("site.back"), backLink, messages("site.back.hiddenText")))) {
+
+    @sectionHeader(messages("liability.sectionHeader"))
+    @pageHeading(messages("liability.taxStartDate.pageHeading"))
+    @paragraphBody(messages("liability.taxStartDate.hint1",
+                                startDate.getDayOfMonth + " " +
+                                messages(s"date.month.${startDate.getMonthValue}") + " " +
+                                startDate.getYear ))
+    @paragraphBody(
+        if(hasExceededThresholdWeight)
+            messages("liability.taxStartDate.ThreshHoldBreached.hint")
```

Comment:
`ThreshHoldBreached` should be `thresholdBreached`

Reply:
yep. I missed that. Thanks.
Done